sql/lockbox_mobile_events_v1.sql

```diff
@@ -0,0 +1,48 @@
+CREATE OR REPLACE VIEW
+`moz-fx-data-derived-datasets.telemetry.lockbox_mobile_events_v1` AS
```

Comment:
We're stuck with "Lockbox" as the app_name string, but seems we should name the view with `lockwise` since we have the flexibility there.

Reply:
good idea